clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -3726,6 +3726,16 @@ RValue CodeGenFunction::EmitBuiltinExpr(const GlobalDecl GD, unsigned BuiltinID,
                             AlignmentCI, OffsetValue);
     return RValue::get(PtrValue);
   }
+  case Builtin::BI__builtin_assume_dereferenceable: {
+    const Expr *Ptr = E->getArg(0);
```

Comment:
Any reason this is a separate variable but the Size one doesn't do this?

Reply:
Pulled Size out to variable as well, thanks!